src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -1185,7 +1185,7 @@ public static void LengthMismatchContains_StringComparison()
 
             string s1 = value.Substring(0, 2);
             string s2 = value.Substring(0, 3);
-            Assert.False(s1.Contains(s2));
+            Assert.DoesNotContain(s2, s1);
```

Comment:
⚠️ Be aware of xunit/xunit#1543 and make sure you don't negatively impact performance

Reply:
For this particular case, I think it *should* be okay since it's a very short string.

Besides, it looks like I made the similar mistake like the `StartsWith`/`EndsWith` here... Will go through the changed bit in `StringTests.cs` again.